script.js

```diff
@@ -1,4 +1,230 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+var gameMode = prompt(
+  " Welcome to a game of Beat that! Please enter your game mode, i.e og, reversed"
+);
+
+var currentMode ='New Game';
+var p1DiceResult =[];
+var p2DiceResult =[];
+var p1DiceResultOrdered =0;
+var p2DiceResultOrdered =0;
+var p1DiceResultOrderedHistory =[];
+var p2DiceResultOrderedHistory =[];
+var p1WinCount =0;
+var p2WinCount =0;
+var p1LoseCount =0;
+var p2LoseCount =0;
+var p1ScoreHistory =[];
+var p2ScoreHistory =[];
+
+//main function
+var main = function(input){
+  if (gameMode == 'og'){
+    return og(input);
+    
+  }
+  if (gameMode =='reversed'){
+    return reversed(input);
+  }
+}
+
+// Dice Roll Function
+var rollDice = function(){
+  ranInt = Math.floor(Math.random() * 6) +1;
+  console.log(ranInt);
+  return ranInt;
+}
+
+//OG game
```

Comment:
Please use more descriptive comments. So that when you come back to it in the future, you will know what these codes suppose to do.